source/basic.tex

```diff
@@ -3989,15 +3989,15 @@
 \rSec3[basic.stc.dynamic]{Dynamic storage duration}%
 
 \rSec4[basic.stc.dynamic.general]{General}%
-\indextext{storage duration!dynamic|(}
```

Comment:
Why is this being removed?

Reply:
The PR adds `\defnadj` within the paragraph below, which presumably obsoletes this.

Reply:
There's a similar thing with a closing paren at the end of the section (I think), which pairs with this one.  You can't just remove one and leave the other. I'd like to point out that remove those parens gets rid of a "n-m" index entry, which we might want to keep.